src/components/_mixins/AccountsMixin.vue

```diff
@@ -93,8 +93,8 @@ export default {
   },
   data() {
     return {
-      availableAccounts: [...Object.keys(EXTERNAL_ACCOUNTS)].filter((account) =>
-        ENABLED_ACCOUNTS.includes(account),
+      availableAccounts: ENABLED_ACCOUNTS.filter(
+        (account) => Object.keys(EXTERNAL_ACCOUNTS).indexOf(account) !== -1,
```

Comment:
Why swap this? And if, this way it's easier to read and faster.
```suggestion
        (account) => account in EXTERNAL_ACCOUNTS,
```

Reply:
Sorry, I get the swap here. My bad. Then just the other thing.

Reply:
Yea, the swap is important, but account in EXTERNAL_ACCOUNTS is definitely the better way to do that.

Reply:
Completed!